packages/client/components/ParabolScopingSearchResultItem.tsx

```diff
@@ -140,12 +140,7 @@ const ParabolScopingSearchResultItem = (props: Props) => {
           addTaskChild('root')
         }}
       >
-        <TipTapEditor
-          editor={editor}
-          linkState={linkState}
-          setLinkState={setLinkState}
-          useLinkEditor={() => useTaskChild('editor-link-changer')}
-        />
+        <TipTapEditor editor={editor} useLinkEditor={() => useTaskChild('editor-link-changer')} />
```

Comment:
@mattkrick We get linter warnings for this one because we pass hooks in a callback. I think it should work if we're careful, but it seems to be not a good idea and not a pattern I would like to keep.